src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -708,6 +745,15 @@ class ErgoWalletActor(settings: ErgoSettings,
     }
   }.flatten
 
+  /**
+    * A helper method that returns unspent boxes
+    */
+  private def getBoxesToSpend: Iterator[TrackedBox] = {
```

Comment:
Better to use parentheses , i.e. getBoxesToSpend() as the function has side-effects